### PR TITLE
Update prevent-changes-to-security-settings-with-tamper-protection.md

### PR DESCRIPTION
**Pull request by anvascon**
Customers are confused about the fact that TP will change settings to their defaults and will then lock them. This will ensure that customers understand that even if they have specific settings/profiles in things like MEM, those settings won't be applied, because they might be against the secure defaults that TP will set.

### CODE REVIEW

**Comment by yogkumgit** [comment]
@JohanFreelancer9 - kindly do the copy editor review for this PR. Thanks

File: microsoft-365/security/defender-endpoint/prevent-changes-to-security-settings-with-tamper-protection.md [hunk 1]
```diff
@@ -47,7 +47,7 @@ With tamper protection, malicious apps are prevented from taking actions such as
 
 ### How it works
 
-Tamper protection essentially locks Microsoft Defender Antivirus and prevents your security settings from being changed through apps and methods such as:
+Tamper protection essentially locks Microsoft Defender Antivirus, to its secure default values, and prevents your security settings from being changed through apps and methods such as:
```
Comment (JohanFreelancer9) [inline microsoft-365/security/defender-endpoint/prevent-changes-to-security-settings-with-tamper-protection.md]:
```suggestion
Tamper protection essentially locks Microsoft Defender Antivirus to its secure default values and prevents your security settings from being changed through apps and methods such as:
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review - @anvascon, kindly commit the minor suggestion. Many thanks.

**Review by denisebmsft — APPROVED** [review]
Thank you so much @anvascon @JohanFreelancer9 @yogkumgit

**Pull request merged**